src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
If I understand correctly, these tests are verifying that all the result data stays within a bound, but they don't verify the correctness of the predictions.  Is that understanding correct?  Is there expected result data (either published, or that could be acquired by running the R functions) that could be checked to make sure they're producing results that are sensible?

Similarly, are the fitted model coefficients close to the coefficients produced by other packages?